shape_optimization/11_Shape_Update_Optimization_Stanford_Bunny/README.md

```diff
@@ -0,0 +1,43 @@
+# Shape Optimization
+Shape update of a wrapping surface to generate a smooth representation of a complex object, which in this example is a Stanford bunny.
+
+> **Author**: Armin Geiser
+>
+> **Kratos version**: 9.0
+
+## Optimization Problem
+The shape update optimization is carried out for two distinct wrapping surfaces: Large sphere and Small sphere
+
+### Objective
+- Large sphere: shape_update minimization 
+- Small sphere: shape_update maximization
+
+### Constraints
+- No penetration of packaging (bounding) mesh (*standford bunny*)
+
+  <p align="center">
+    <img src="images/bunny_opt_setup.png" height="500">
```

Comment:
please make the annotated text smaller, it should be approx the font size of the normal markdown text